### PR TITLE
Register ConicIP 

### PR DESCRIPTION
**Pull request by gabgoh**
Another try ! https://github.com/JuliaLang/METADATA.jl/pull/5981

### CODE REVIEW

File: ConicIP/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4 
+MathProgBase 0.5 0.7
+WoodburyMatrices
+Compat
```
Comment (tkelman) [inline ConicIP/versions/0.0.1/requires]: It's best to declare a minimum version dependency that implements the features you need.

**Comment by gabgoh** [comment]
Would this work?

```
julia 0.5
MathProgBase 0.5 0.6
MathProgBase 0.5 0.7
WoodburyMatrices 0.2
Compat 0.21
```

File: ConicIP/versions/0.0.1/requires (hunk 1 above)
Comment (gabgoh) [inline ConicIP/versions/0.0.1/requires]: Would this work? It is a little conservative

```
julia 0.5
MathProgBase 0.5 0.6
MathProgBase 0.5 0.7
WoodburyMatrices 0.2
Compat 0.21
```

Comment (tkelman) [inline ConicIP/versions/0.0.1/requires]: that's too tight. what features are you using from compat?

Comment (gabgoh) [inline ConicIP/versions/0.0.1/requires]: It seems I don't need compat after all! I'm removing all references, and reverting to

```
julia 0.5
MathProgBase 0.5 0.7
WoodburyMatrices 0.2
```

Comment (tkelman) [inline ConicIP/versions/0.0.1/requires]: that looks fine

**Comment by gabgoh** [comment]
Closing this pull request, new one submitted with all relevant changes

**Pull request closed without merging**